packages/tpetra/CMakeLists.txt

```diff
@@ -172,6 +172,21 @@ ELSE ()
       ENDIF ()
     ENDIF ()
 
+    IF ((NOT Tpetra_FOUND_OMPI_INFO_EXECUTABLE) AND (DEFINED MPI_USE_COMPILER_WRAPPERS) AND MPI_USE_COMPILER_WRAPPERS AND (DEFINED CMAKE_CXX_COMPILER))
```

Comment:
This looks like an exact duplicate of the logic above.  Am I missing something, or does CMake weirdly need this to work?

Reply:
The check here is for CMAKE_CXX_COMPILER rather than MPI_CXX_COMPILER; that's the only difference.

Reply:
@kddevin Thanks for clarifying :D